src/Uno.UI/UI/Xaml/Controls/Layouter/Layouter.Android.cs

```diff
@@ -58,6 +58,17 @@ protected Size MeasureChildOverride(View view, Size slotSize)
 			var widthSpec = ViewHelper.SpecFromLogicalSize(slotSize.Width);
 			var heightSpec = ViewHelper.SpecFromLogicalSize(slotSize.Height);
 
+			if (double.IsPositiveInfinity(slotSize.Width) || double.IsPositiveInfinity(slotSize.Height))
+			{
+				// Bypass Android cache, to ensure the Child's Measure() is actually invoked.
+				view.ForceLayout();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'view' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5255139)